chain/network/Cargo.toml

```diff
@@ -20,6 +20,8 @@ byteorder = "1.2"
 lazy_static = "1.4"
 tracing = "0.1.13"
 strum = { version = "0.20", features = ["derive"] }
+near-rust-allocator-proxy = "0.2.9"
+size = "0.1.2"
```

Comment:
Hm, why are we using both `size` and `bytesize` crates? They seem to do essentially the same thing?

Reply:
I agree, we should use `bytesize` to be consistent. I would like to use `ByteInfo` global constants, the PR is submitted and will be included in next release of `bytesize` crate. Once it gets published to `crates.io`, it would be a good time to replace `size.